tests/ArraySize/Makefile.in

```diff
@@ -1,2 +1,3 @@
 TOP_FILE := $(TEST_DIR)/top.sv
 TOP_MODULE := top
+USE_YOSYS_SURELOG_FRONTEND ?= 1
```

Comment:
I think we don't really need to define it in the test ``Makefile.in`` and it should be set by CI/when executing ``make``, or I'm missing something?

Reply:
I wanted to make this test parsed by surelog frontend by default, but leave the other tests to be parsed by uhdm frontend.

Reply:
I didn't want to modify the ci script for this test only, but I think it will be helpful to have at least one test that uses different frontend.